src/frontend/screens/ObservationEdit/index.tsx

```diff
@@ -52,15 +52,19 @@ export const ObservationEdit: NativeNavigationComponent<'ObservationEdit'> = ({
   route,
 }) => {
   const {formatMessage} = useIntl();
-  const {projectApi} = useActiveProject();
+  const {projectApi, projectId} = useActiveProject();
 
   const value = usePersistedDraftObservation(store => store.value);
   const {updateTags, clearDraft, usePreset, existingObservationToDraft} =
     useDraftObservation();
   const preset = usePreset();
-  const editObservationMutation = useEditObservation();
+  const {mutate, status, reset, error} = useUpdateDocument({
+    docType: 'observation',
+    projectId,
+  });
   const attachments = usePersistedDraftObservation(store => store.attachments);
-  const createBlobMutation = useCreateBlobMutation();
+  const {mutateAsync: createBlobAsync, reset: resetAttachment} =
```

Comment:
update name to be more accurate:

```suggestion
  const {mutateAsync: createAttachmentAsync, reset: resetAttachment} =
```